replication/row_event.go

```diff
@@ -1120,6 +1120,18 @@ func (e *RowsEvent) Decode(data []byte) error {
 	return e.DecodeData(pos, data)
 }
 
+func (e *RowsEvent) IsInsert() bool {
```

Comment:
Yes currently `RowsEvent` has no public type information. You can know the type from `EventHeader`